lib/facter/apt_updates.rb

```diff
@@ -42,6 +42,17 @@
   end
 end
 
+Facter.add("apt_package_security_updates") do
```

Comment:
How is this different to lines 63-67, apologies if i am missing something obvious. No caffeine yet.